openshift/topics/advanced_concepts.adoc

```diff
@@ -405,6 +405,68 @@ When restarting the server it is important to restart just the JBoss EAP process
 ====
 . Log in to the `master` realm's administration console of the {project_name} server using the credentials created in the steps above. In the browser, navigate to *\http://sso-<project-name>.<hostname>/auth/admin*  for the {project_name} web server, or to *\https://secure-sso-<project-name>.<hostname>/auth/admin* for the encrypted {project_name} web server, and specify the user name and password used to create the administrator user.
 
+
+=== Customizing the default behavior of the {project_name} image
+
+You can change the default behavior of the {project_name} image such as enabling TechPreview features or enabling debugging. This section describes how to make this change by using the JAVA_OPTS_APPEND variable.
```

Comment:
*TechPreview* is never used across the documentation. This should be replaced with *technology preview* in full